src/styles/carousel.css

```diff
@@ -0,0 +1,60 @@
+/* Carousel */
```

Comment:
where did the style for this come from?  is just mostly overriding the default bootstrap carousel stuff?

Reply:
I did extract only the CSS parts that I needed for the carousel, without including the entire bootstrap CSS file (which would have messed up the style of the website)